UE4SS/src/GUI/Dumpers.cpp

```diff
@@ -83,21 +83,21 @@ namespace RC::GUI::Dumpers
     {
         StringType root_actor_buffer{};
 
-        static auto location_property = root_component->GetPropertyByNameInChain(STR("RelativeLocation"));
-        static auto rotation_property = root_component->GetPropertyByNameInChain(STR("RelativeRotation"));
-        static auto scale_property = root_component->GetPropertyByNameInChain(STR("RelativeScale3D"));
+        static auto location_property = root_component->GetPropertyByNameInChain(FromCharTypePtr<TCHAR>(STR("RelativeLocation")));
```

Comment:
We should have documentation on when to use this cast.

Reply:
So this function is for testing purposes only until Linux support has been merged, and after that the function can be removed.

See this and the comment below it: https://github.com/Re-UE4SS/UEPseudo/pull/96#issuecomment-2174994211

Reply:
As I understand, `FromCharTypePtr` should be used now for whenever we convert a char pointer to unreal character pointer. And this function will only work if those are the same size so no conversion would be needed. And this would be removed by the time linux support gets introduced?

Reply:
> As I understand, `FromCharTypePtr` should be used now for whenever we convert a char pointer to unreal character pointer. And this function will only work if those are the same size so no conversion would be needed. And this would be removed by the time linux support gets introduced?

Yes, and it would likely be removed either by the linux support PR (at which point CI takes over responsibility to make sure the Linux build doesn't break) or soon after if it gets forgotten.